test/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter_test.cc

```diff
@@ -250,6 +251,25 @@ TEST_F(GrpcHttp1BridgeFilterTest, HandlingBadGrpcStatus) {
   EXPECT_EQ("foo", response_headers.get_("grpc-message"));
 }
 
+TEST_F(GrpcHttp1BridgeFilterTest, HandlingNoTailerBadGrpcStatus) {
```

Comment:
Maybe call this `HandlingHeadersOnlyResponseBadGrpcStatus`? I think "headers only" is a more common way to refer to this kind of responses when talking about gRPC

Reply:
good name